modules/claims_api/lib/claims_api/special_issue_mappers/bgs.rb

```diff
@@ -0,0 +1,167 @@
+          { name: 'RO Special Issue 7', code: 'RSI7' },
+          { name: 'RO Special Issue 8', code: 'RSI8' },
+          { name: 'RO Special Issue 9', code: 'RSI9' },
+          { name: 'RVSR Examination', code: 'REN' },
+          { name: 'Radiation', code: 'RDN' },
+          { name: 'Radiation Radiogenic Disability Confirmed', code: 'RRDC' },
+          { name: 'Rating Decision Review - Level 1', code: 'RDR1' },
+          { name: 'Rating Decision Review - Level 2', code: 'RDR2' },
+          { name: 'Ready for Exam', code: 'RFE' },
+          { name: 'Same Station Review', code: 'SSR' },
+          { name: 'SHAD', code: 'SHAD' },
+          { name: 'Sarcoidosis', code: 'SARCO' },
+          { name: 'Simultaneous Award Adjustment Not Permitted', code: 'SAANP' },
+          { name: 'Specialized Records Request', code: 'SPECRECRQST' },
+          { name: 'Stage 1 Development', code: 'S1D' },
+          { name: 'Stage 2 Development', code: 'S2D' },
+          { name: 'Stage 3 Development', code: 'S3D' },
+          { name: 'TBI Exam Review', code: 'TER' },
+          { name: 'Temp 100 Convalescence', code: 'TMC' },
+          { name: 'Temp 100 Hospitalization', code: 'T1H' },
+          { name: 'Tobacco', code: 'TOB' },
+          { name: 'Tort Claim', code: 'TC' },
+          { name: 'Traumatic Brain Injury', code: 'TBI' },
+          { name: 'Upfront Verification', code: 'UV' },
+          { name: 'VACO Special issue 1', code: 'COSI1' },
+          { name: 'VACO Special issue 2', code: 'COSI2' },
+          { name: 'VACO Special Issue 3', code: 'VACSPISTHR' },
+          { name: 'VACO Special Issue 4', code: 'VACSPISFOR' },
+          { name: 'VACO Special Issue 5', code: 'VACSPIS5' },
+          { name: 'VACO Special Issue 6', code: 'VACSPIS6' },
+          { name: 'VACO Special Issue 7', code: 'VACSPIS7' },
+          { name: 'VACO Special Issue 8', code: 'VACSPIS8' },
+          { name: 'VACO Special Issue 9', code: 'VACSPIS9' },
+          { name: 'VASRD-Cardiovascular', code: 'VC' },
+          { name: 'VASRD-Dental', code: 'VD' },
+          { name: 'VASRD-Digestive', code: 'VRD' },
+          { name: 'VASRD-Endocrine', code: 'VEN' },
+          { name: 'VASRD-Eye', code: 'VE' },
+          { name: 'VASRD-GU', code: 'VG' },
+          { name: 'VASRD-GYN', code: 'VDG' },
+          { name: 'VASRD-Hemic', code: 'VH' },
+          { name: 'VASRD-Infectious', code: 'VI' },
+          { name: 'VASRD-Mental', code: 'VML' },
+          { name: 'VASRD-Musculoskeletal', code: 'VM' },
+          { name: 'VASRD-Neurological', code: 'VN' },
+          { name: 'VASRD-Respiratory/Auditory', code: 'VRA' },
+          { name: 'VASRD-Skin', code: 'VS' },
+          { name: 'Vendor Exclusion - No Diagnosis', code: 'VEXCLNODIAG' },
+          { name: 'VONAPP Direct Connect', code: 'VDC' },
+          { name: 'WARTAC', code: 'WARTAC' },
+          { name: 'WARTAC Trainee', code: 'WT' }
+        ] + super).uniq { |special_issue| special_issue[:name] }
```

Comment:
Should we freeze this collection like we do in the parent?

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/6014/files#diff-a09bad8abe02f0b3548627a2840903b06a8424e211cbd9980773b6108ed55269R163